src/rules/indentRule.ts

```diff
@@ -50,7 +50,13 @@ class IndentWalker extends Lint.RuleWalker {
         }
         var scanner = ts.createScanner(ts.ScriptTarget.ES5, false, node.text);
         var lineStarts = node.getLineStarts();
+        var endOfComment = -1;
         lineStarts.forEach((lineStart) => {
+            if (lineStart < endOfComment) {
+                // Skip checking lines inside multi-line comments
```

Comment:
shouldn't TSLint have complained here since we turned on the comment-format rule?